client/src/components/Entries.jsx

```diff
@@ -17,9 +17,9 @@ export default class Entries extends Component {
       <div className="eight wide column">
         {entries.map(entry => 
           <Entry 
-            date={entry.date} 
+            date={entry.created} 
             text={entry.text} 
-            key={entry.date}
+            key={Math.random()}
```

Comment:
Couldn't the key also be entry.created given that we only allow 1 entry to be created? Otherwise we can just use entry.entry_id.